src/js/models/card-node.js

```diff
@@ -35,7 +35,8 @@ export default class CardNode {
         this.display();
       },
       cancel: () => this.display(),
-      remove: () => this.remove()
+      remove: () => this.remove(),
+      card: this
```

Comment:
`this` is the card node, not the instance of the card as a user sees it (`this.card[mode]` I believe). I'm not sure making the `CardNode` public API for card implementers is ideal.